Automattic/wp-calypso
# Launchpad: Remove useLaunchpadChecklist hook

Related to #76597

## Proposed Changes

* Remove unused `useLaunchpadChecklist` hook; it's been deprecated in favor of `useLaunchpad` and there are no instances of it in the codebase.

<img width="453" alt="Screen Shot 2023-05-23 at 4 23 40 PM" src="https://github.com/Automattic/wp-calypso/assets/2124984/7c25c0bc-7614-4c95-8e09-003c44064121">

## Testing Instructions



* Switch to this PR
* Create a free site from `/start` with the Write & Publish intent.
* Skip to dashboard to see the Launchpad in Stepper
* Ensure Launchpad still works as expected.
* Skip to dashboard again to go to the site preview/Calypso area.
* Check inline help (screenshot above); the Launchpad indicator here should still work with no errors in the console.

## Pre-merge Checklist
